ci/run_integration_test.sh

```diff
@@ -42,5 +42,5 @@ echo "export SCYLLA_VERSION=${SCYLLA_RELEASE}"
 echo "PROTOCOL_VERSION=4 EVENT_LOOP_MANAGER=asyncio pytest --import-mode append tests/integration/standard/"
 export SCYLLA_VERSION=${SCYLLA_RELEASE}
 export MAPPED_SCYLLA_VERSION=3.11.4
-PROTOCOL_VERSION=4 EVENT_LOOP_MANAGER=libev pytest -rf --import-mode append $*
```

Comment:
It depends on which pytest version we are using I think it needs to be 7.0 an up